doc/rest.json

```diff
@@ -377,6 +377,9 @@
                                         },
```

Comment:
in the data I see: id, status, report

Reply:
$result['id']     = $this->getTaskId($taskLogEntity);
        $result['status'] = $this->getTaskStatus($taskLogEntity);
        $result['report'] = $taskLogEntity->getReport() ? $this->getTaskReport($taskLogEntity) : [];

        return array_merge($result, (array) $this->addExtraReturnData($taskLogEntity));

Reply:
addExtraReturnData adds the field `itemIds`

Reply:
And that's strange that swagger editor didn't highlight the error for you. For me it's: ```Semantic error at paths./taoQtiItem/RestQtiItem/getStatus.get.responses.200.schema.properties.data.properties.itemIds
Schemas with 'type: array', require a sibling 'items: ' field
Jump to line 377
```